mantidimaging/core/rotation/test/polyfit_correlation_test.py

```diff
@@ -4,6 +4,7 @@
 import unittest
 from unittest import mock
 import numpy as np
+from numpy.random._examples.cffi.extending import rng
```

Comment:
I think this is the wrong `rng` to use, so the line can be remove.

Then use `rng = np.random.default_rng()` where needed.